Automattic/wp-calypso
# Launchpad: Allow users to revisit completed steps

#### Summary
Currently, once the user lands on the Launchpad checklist screen via the Newsletter flow, they are unable to go back to the **Add Subscribers** task since it's disabled:

![cUk0Tf.png](https://user-images.githubusercontent.com/20927667/188510845-a38ade0f-ef85-4dd5-8a7b-9bf787031ca5.png)

#### Proposed changes
This PR allows completed tasks to be kept active (currently all completed tasks are disabled) so that the user could revisit a completed task and make changes. In particular, we are enabling the **"Add Subscribers"** task:

![IFbMN3.png](https://user-images.githubusercontent.com/20927667/188736305-04938cba-baa1-4cf0-b65a-b355419a5642.png)

#### Testing Instructions
1. Go to `http://calypso.localhost:3000/setup/launchpad?flow=newsletter&complete-setup=true&siteSlug=YOUR_SITE_SLUG`
2. You should see that the **"Add Subscribers"** task is completed but still active
3. Click on the **"Add Subscribers"** task
4. You should see the **"Add subscribers to build your audience"** screen

https://user-images.githubusercontent.com/20927667/188726996-506b2bdf-9864-4a30-a0ba-d2b7d1d0a737.mov

#### Pre-merge Checklist






Fixes #67336
